.upsun/config.yaml

```diff
@@ -23,7 +23,7 @@ applications:
         hooks:
             build: |
                 set -eux
-                npm install
+                npm install --silent
```

Comment:
The `npm_config_loglevel` should already do this @tylers-username

Reply:
Ah, I didn't realize that had already been included. 

I attempted `npm install --log-level=error` but continued to see the vulnerability warnings

Reply:
If the build hook no longer shows warnings we can close this. Sorry for the noise 😄

Reply:
`main` seems alright. 

<img width="797" alt="image" src="https://github.com/platformsh/demo-project/assets/4887071/d488dda3-337a-4f50-ab88-3b3862220d27">


I'll close.